tests/test_filter_job_by_email.py

```diff
@@ -30,4 +30,4 @@ def test_remove_email_address_filter(base_url, selenium):
     page.click_on_active_watched_repo()
     all_emails = [email.get_name for email in page.all_emails]
 
-    assert len(set(all_emails)) > 1
+    assert len(all_emails) > 1
```

Comment:
Changing this to a `set` meant that duplicates were removed. Filtering by an email address may return multiple commits for the same email address, so this assert can now cause a false positive.